lib/plugins/aws/package/index.js

```diff
@@ -62,6 +64,9 @@ class AwsPackage {
       'package:setupProviderConfiguration': () => BbPromise.bind(this)
         .then(this.mergeIamTemplates),
 
+      'after:package:setupProviderConfiguration': () => BbPromise.bind(this)
+        .then(this.validateCompiledTemplate),
```

Comment:
The validate template step should not be part of the `package` lifecycle, as this would force build servers to have direct access to AWS REST endpoints. Normally only deployment servers in a CI/CD system have access to internet resources.

It should just be added to  the implementation of `aws:deploy:updateStack` where also the upload is located.